src/os/bluestore/NVMEDevice.cc

```diff
@@ -795,6 +795,10 @@ void NVMEDevice::close()
 {
   dout(1) << __func__ << dendl;
 
+  if (queue_t) {
```

Comment:
nit, no need to check for nullptr.

Reply:
@optimistyzy why not making `queue_t` a member variable of `NVMEDevice` ? as it holds a reference of the `NVMEDevice`. once the device is closed or destroyed, the `queue_t` will be holding a dangling reference. what do you think?

Reply:
@tchaikov, thanks! I will remove the check of "queue_t".